tests/test_ignite.py

```diff
@@ -0,0 +1,11 @@
+import chainer_pytorch_migration.ignite
+
+
+def test_version_comparisson():
+    version_fnct = chainer_pytorch_migration.ignite._get_version
+    assert version_fnct('0.0.0') < version_fnct('0.3.0')
```

Comment:
How about testing like `assert version_fnct('0.0.0') == [0, 0]` but I'm not sure this function needs a test.

Reply:
Fixed!
I also changed the version function to use a regular expression and added some invalid case tests too. But yeah, I think that testing this may be a bit overkill